lib/app/dependency.go

```diff
@@ -126,13 +139,35 @@ func getDependencies(app *Application, apps Applications, state *state) error {
 	if runtimePackage, _ := app.Manifest.DefaultRuntimePackage(); runtimePackage != nil {
 		state.runtimePackage = runtimePackage
 	}
-
 	return nil
 }
 
+func (r *state) isPackage(pkg loc.Locator) bool {
```

Comment:
Nit: I think `hasPackage` would be a bit more appropriate name for this method. Same for `hasApp` below.